### ISSUE TITLE
Test against Python 3.10

### ISSUE THREAD

**Issue by jakebeal** [iGEM-Engineering/iGEM-distribution#310]
We are currently running up through Python 3.9. Some things that were deprecated have finally vanished in Python 3.10, and the scripts may not work correctly for that version.

**Comment by mburridge96** [comment]
Biopython seems to have some build issues with python3.10 on mine - x86_64-linux-gnu-gcc error 

It doesn't seem to be used anywhere though, so should biopython be removed from requirements? Or am I missing something?

EDIT: Solved that, need both build-essential and python dev installed.

**Comment by noahsprent** [comment]
How does this issue relate to #180? Can we test for 3.7-3.10 like SBOL-utilities?

**Comment by jakebeal** [comment]
Yes, we could upgrade to a matrix build. The SBOL utilities script shows how to do this.

**Comment by jakebeal** [comment]
Doing a matrix build would simultaneously address both this and #180
